app/controllers/admin/users_controller.rb

```diff
@@ -10,6 +10,19 @@ def index
       @users = User.all
     end
 
+    def toggle_confirmation
+      user = User.find(params[:id])
+
+      if params[:value] == "true"
+        user.confirm
+        user.save
+      elsif params[:value] == "false"
+        user.revoke
```

Comment:
This is hard to understand when reading. 
It is also a bit misleading because we are using rolify gem that uses `user.revoke`.  https://github.com/RolifyCommunity/rolify/wiki
I think you should just do whatever you need to do in the controller.